### PR TITLE
Add slot_height() and epoch_height() methods to Bank

### PR DESCRIPTION
**Pull request by garious**
#### Problem

LeaderSchedule piggybacks off calls to Bank::register_tick() to update its internal copy of `tick_height` so that it can calculate slot height and epoch height at any given moment.

#### Summary of Changes

Make that data available available from the Bank.

Note: this initial version assumes static configuration.

@carllin, @aeyakovenko, fyi

### CODE REVIEW

File: runtime/src/bank.rs [hunk 1]
```diff
@@ -606,10 +609,31 @@ impl Bank {
             .collect()
     }
 
+    /// Return the number of ticks since genesis.
     pub fn tick_height(&self) -> u64 {
         self.last_id_queue.read().unwrap().tick_height
     }
 
+    /// Return the number of ticks since the last slot boundary.
+    pub fn tick_index(&self) -> u64 {
```
Comment (mvines) [inline runtime/src/bank.rs]: What’s the plan to remove these constants?

Comment (garious) [reply runtime/src/bank.rs]: The first step is to add them to GenesisBlock and use them to set member variables in Bank. Eventually, they should be updated as we process the ledger and observe votes on cluster parameters.

Want me to add the GenesisBlock part as part of this PR or add it just after?

Comment (mvines) [reply runtime/src/bank.rs]: After is fine with me, I just wanted to ensure you weren’t operating under the assumption that those defaults are fixed.

Comment (garious) [reply runtime/src/bank.rs]: I'll update `LeaderScheduler::new_with_bank()` to allow these to comfortably coexist as we transition more to `LeaderSchedule::new_with_bank()` (note the missing R)

**Pull request merged**